### PR TITLE
Replace short license with full text of license

### PR DESCRIPTION
**Pull request by drassaby**
GitHub recognizes the full text and has helpful breakdowns of permissions, like this:
</br>

License Permissions Breakdown

</br>

It will also show up in the top bar like this:
</br>
GitHub top bar with license

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,13 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (drewinglis) [inline LICENSE]: Need to fill this out.

**Review by drewinglis — DISMISSED** [review]

File: LICENSE (hunk 1 above)
Comment (drassaby) [reply LICENSE]: Fixed!

**Review by drewinglis — APPROVED** [review]
I think this is fine? Not sure if we should continue to include the (C) Amperity at the top of the file also...?

**Comment by drassaby** [comment]
Shouldn't matter, won't change the actual license agreement either way 👍, which requires license/copyright notice

**Pull request merged**